### DISCUSSION TITLE
Creating CalendarView with .NET MAUI Handler architecture

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#87] [Announcements]
# Creating CalendarView with .NET MAUI Handler architecture

Explore the creation of a CalendarView control in .NET MAUI using Handler architecture.

https://vladislavantonyuk.github.io/articles/Creating-CalendarView-with-.NET-MAUI-Handler-architecture/

**Comment by cyrillgisin** [comment]
Good day

Could you please make a tutorial how to implement a context-menu like this in ios and android: https://docs.devexpress.com/MAUI/images/popup/popup-context-menu.png
Important here is that the context-menu appears next to the element (s.a. a button or an image) and automatically closes when clicking elsewhere.

Best regards
